### PR TITLE
[Docs] Update getting_started.md to remove elastic reference and other changes

### PR DESCRIPTION
**Pull request by manasvinibs**
### Issues Resolved
https://github.com/opensearch-project/oui/issues/104
 
### Check List

- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src-docs/src/views/guidelines/getting_started.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ yarn add @opensearch-project/oui
 Note that OUI has [several `peerDependencies` requirements](package.json) that will also need to be installed if starting with a blank project. You can read more about other ways to [consume OUI][consuming].
 
 ```js
-yarn add @opensearch-project/oui @elastic/datemath moment prop-types
+yarn add @opensearch-project/oui @opensearch/datemath moment prop-types
```
Comment (kavilla) [inline src-docs/src/views/guidelines/getting_started.md:12]: I don't think this is true: https://github.com/opensearch-project/oui/blob/main/package.json#L135

Comment (manasvinibs) [reply src-docs/src/views/guidelines/getting_started.md:12]: That's right! Thanks Rocky for verifying. As OUI is still referencing @elastic/datemath at this point and have not updated it to @opensearch/datemath, I'll keep elastic references as it is in the code for now. cc @BSFishy

**Review by seanneumann — DISMISSED** [review]
LGTM

**Review by BSFishy — CHANGES_REQUESTED** [review]
There are still references to Kibana on this page

**Review by BSFishy — APPROVED** [review]

File: src-docs/src/views/guidelines/getting_started.md (hunk 1 above)
Comment (BSFishy) [reply src-docs/src/views/guidelines/getting_started.md]: Fixed in #204

**Review by seanneumann — APPROVED** [review]
👍

**Pull request merged**